modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/execution/ReflectiveOperationExecutorFactory.java

```diff
@@ -46,7 +47,7 @@ public ReflectiveOperationExecutorFactory(Class<T> implementationClass, Method o
 
   @Override
   public ComponentExecutor<M> createExecutor(M operationModel, Map<String, Object> parameters) {
-    DefaultObjectBuilder objectBuilder = new DefaultObjectBuilder(implementationClass);
+    DefaultObjectBuilder objectBuilder = new DefaultObjectBuilder(implementationClass, new ReflectionCache());
```

Comment:
why create a new cache here instead of getting the one from the registry?

Reply:
Registry doesn't exist at this point. This is just to create an internal object that will never be seen again.